config.default.yaml

```
@@ -657,7 +658,20 @@ sector:
   efficiency_heat_gas_to_elec: 0.9
 
   electricity_distribution_grid: true # adds low voltage buses and shifts AC loads, BEVs, heat pumps, and resistive heaters, micro CHPs to low voltage buses if technologies are present
-  solar_rooftop: true # adds distribution side customer rooftop PV (only work if electricity_distribution_grid: true)
+  solar_rooftop: # adds distribution side customer rooftop PV (only work if electricity_distribution_grid: true)
+    enable: true
+    kW_per_m2: 0.1
+    m2_per_person: 20
+    use_building_size: false
+    install_ratio:
```

Not sure `install_ratio` is crystal clear. Could it make sense to add a clarification comment for that? E.g. we can directly use a definition from Hideaki Obane’s paper:

```suggestion
    # proportion of rooftop suitable for PV installation
    install_ratio:
```